### PR TITLE
comp ids

### PR DESCRIPTION
**Pull request by scopatz**
I stupidly named two of my computers with the same hostname, which only drew out this issue.

### CODE REVIEW

File: news/compid.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Added:** None
+
+**Changed:**
+
+* GitHub token notes now have unique identifiers, which prevents issues from arrising
```
Comment (CJ-Wright) [inline news/compid.rst]: I think its arising

**Comment by CJ-Wright** [comment]
otherwise lgtm

File: news/compid.rst (hunk 1 above)
Comment (scopatz) [reply news/compid.rst]: You rr correct!

**Comment by scopatz** [comment]
looks like it passes

**Comment by CJ-Wright** [comment]
to be fair I don't know if we test that chunk of the code

**Comment by scopatz** [comment]
This chunk of code is hard to test because it involves making and destroying github tokens. So either the test code would need a user's password or we'd need need a separate test account with a semi-secret password.  Also, all of that is presumably done in the tests of github3.py.

**Comment by scopatz** [comment]
thanks for the merge!

**Comment by CJ-Wright** [comment]
Yea, I mean my point was more that I didn't expect the PR to fail.
Sure!

**Pull request merged**